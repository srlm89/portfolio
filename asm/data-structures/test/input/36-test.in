ERASE actual/36-test.out
INSERT 30 4.645160 Madagascar 2 Escape 2 Africa [2008] [720p]
PRINT_IN_ORDER actual/36-test.out
INSERT 31 6.296640 Darko 720P
INSERT 12 7.920039 Race To Witch Mountain [1080p]
REMOVE_ID 30
PRINT_IN_ORDER actual/36-test.out
INSERT 6 5.079679 Crossing Over 720P
INSERT 7 5.554938 Crash    1080p
INSERT 40 5.920277 Star Wars Saga  Collection 720p
REMOVE_ID 40
PRINT_IN_ORDER actual/36-test.out
INSERT 2 6.286243 Poseidon   1080p
INSERT 4 5.271229 Connected   720p
INSERT 13 9.664683 Alone in the dark   720p
INSERT 16 9.368591 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p]
REMOVE_ID 2
REMOVE_ID 13
PRINT_IN_ORDER actual/36-test.out
INSERT 44 7.665204 The Secret of Moonarce [720p]
INSERT 27 1.509277 Burn E [720p]
INSERT 28 9.355646 Scooby.Doo.The.Mystery.Begins.2009   720p
INSERT 45 5.020922 Jumper    1080p
INSERT 46 6.711421 10,000 BC 720p
REMOVE_ID 28
REMOVE_ID 12
PRINT_IN_ORDER actual/36-test.out
INSERT 20 7.580083 Toy.Story.1995.720p.HDTV.x264-CtrlHD
INSERT 32 9.483675 Firewall 720p
INSERT 53 3.855345 007 Licence to Kill [1989] [BluRay] [720p] [DTS]
INSERT 37 5.532628 CJ7   720p
INSERT 24 9.137203 Babylon A.D    720p
INSERT 51 7.068907 Golden Compass   720p
REMOVE_ID 16
REMOVE_ID 46
REMOVE_ID 45
PRINT_IN_ORDER actual/36-test.out
INSERT 42 2.844396 007 Goldfinger [1964] [BluRay] [720p] [DTS]
INSERT 10 8.237954 Stargate-The Ark of Truth 720p
INSERT 18 0.410772 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS
INSERT 15 5.963848 Terminator Salvation Director's cut  ( 2009)    1080p
INSERT 14 9.770458 The Mist   720p
INSERT 55 9.240480 Meet The Parents    720p
INSERT 19 8.224514 Final Destination 4   1080p
REMOVE_ID 24
REMOVE_ID 20
REMOVE_ID 51
PRINT_IN_ORDER actual/36-test.out
INSERT 47 1.648925 IMAX Ring of Fire [720p] [Documenter]
INSERT 61 5.061433 G Force    720p
INSERT 38 2.008956 007.1983.Octopussy.720p.hdtv.x264
INSERT 33 1.821950 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON
INSERT 60 0.415329 Eragon 720p
INSERT 58 2.167372 Charlie and the Chocolate Factory   720p
INSERT 39 0.849620 Infernal Affair 1-3   1080p
INSERT 71 4.666272 Eurotrip    720p
REMOVE_ID 4
REMOVE_ID 53
REMOVE_ID 39
REMOVE_ID 58
PRINT_IN_ORDER actual/36-test.out
INSERT 35 7.382330 Gladiator 1080p
INSERT 49 5.400412 Ice Age 3 Dawn of the Dinosaurs [2009] [720p]
INSERT 9 0.299931 Daredevil - 720p
INSERT 25 8.433877 Apollo 13   1080p
INSERT 75 7.211695 Cloverfield   1080p
INSERT 70 2.180707 Hancock    1080p
INSERT 48 4.956958 Hulk 1    1080p
INSERT 67 9.335771 Knowing 2009 720p
INSERT 62 8.035188 Harold & Kumar   720p
REMOVE_ID 27
REMOVE_ID 15
REMOVE_ID 37
REMOVE_ID 14
PRINT_IN_ORDER actual/36-test.out
INSERT 65 7.675481 Starship Troopers 1  1080p
INSERT 78 3.125565 Fantastic Four   1080p
INSERT 76 7.626986 Get Smart (2008) 1080p
INSERT 21 8.377139 PBS
INSERT 1 3.427969 Kingdom of Heaven [2005] [720p]
INSERT 59 4.214503 Shoot.Em.Up.720p.Bluray.x264-SEPTiC
INSERT 3 1.768690 My Bloody Valentine 3D    1080p
INSERT 22 5.940340 Jerry Maguire    1080p
INSERT 11 2.345214 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD
INSERT 74 5.510212 High school musical 2006    720p
REMOVE_ID 19
REMOVE_ID 67
REMOVE_ID 38
REMOVE_ID 32
REMOVE_ID 44
PRINT_IN_ORDER actual/36-test.out
INSERT 81 5.442692 Jeniffer's Body ( 2009)    1080p
INSERT 91 7.485249 The Bank Job [720p]
INSERT 83 0.188823 PIXAR Movie
INSERT 87 8.043926 Hero    1080p
INSERT 5 5.917003 Yes Man   720p
INSERT 93 7.621348 Rocky 2 [1979] [BluRay] [720p] [AC3]
INSERT 88 2.596885 Lord Of The Ring  - Return Of The King   720p
INSERT 56 2.962803 Afro samurai uncut 720p
INSERT 57 9.234597 Max Payne    720p
INSERT 100 9.749040 The Sniper 720p
INSERT 52 6.428285 American Beauty   720p
REMOVE_ID 52
REMOVE_ID 55
REMOVE_ID 71
REMOVE_ID 65
REMOVE_ID 57
PRINT_IN_ORDER actual/36-test.out
