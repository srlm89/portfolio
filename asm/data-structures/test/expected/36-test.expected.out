[ 30 4.645160 Madagascar 2 Escape 2 Africa [2008] [720p] ]
[ 12 7.920039 Race To Witch Mountain [1080p] ][ 31 6.296640 Darko 720P ]
[ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 12 7.920039 Race To Witch Mountain [1080p] ][ 31 6.296640 Darko 720P ]
[ 4 5.271229 Connected   720p ][ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 12 7.920039 Race To Witch Mountain [1080p] ][ 16 9.368591 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 31 6.296640 Darko 720P ]
[ 4 5.271229 Connected   720p ][ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 16 9.368591 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 27 1.509277 Burn E [720p] ][ 31 6.296640 Darko 720P ][ 44 7.665204 The Secret of Moonarce [720p] ][ 45 5.020922 Jumper    1080p ][ 46 6.711421 10,000 BC 720p ]
[ 4 5.271229 Connected   720p ][ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 20 7.580083 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 24 9.137203 Babylon A.D    720p ][ 27 1.509277 Burn E [720p] ][ 31 6.296640 Darko 720P ][ 32 9.483675 Firewall 720p ][ 37 5.532628 CJ7   720p ][ 44 7.665204 The Secret of Moonarce [720p] ][ 51 7.068907 Golden Compass   720p ][ 53 3.855345 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ]
[ 4 5.271229 Connected   720p ][ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 10 8.237954 Stargate-The Ark of Truth 720p ][ 14 9.770458 The Mist   720p ][ 15 5.963848 Terminator Salvation Director's cut  ( 2009)    1080p ][ 18 0.410772 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 19 8.224514 Final Destination 4   1080p ][ 27 1.509277 Burn E [720p] ][ 31 6.296640 Darko 720P ][ 32 9.483675 Firewall 720p ][ 37 5.532628 CJ7   720p ][ 42 2.844396 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 44 7.665204 The Secret of Moonarce [720p] ][ 53 3.855345 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 55 9.240480 Meet The Parents    720p ]
[ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 10 8.237954 Stargate-The Ark of Truth 720p ][ 14 9.770458 The Mist   720p ][ 15 5.963848 Terminator Salvation Director's cut  ( 2009)    1080p ][ 18 0.410772 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 19 8.224514 Final Destination 4   1080p ][ 27 1.509277 Burn E [720p] ][ 31 6.296640 Darko 720P ][ 32 9.483675 Firewall 720p ][ 33 1.821950 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 37 5.532628 CJ7   720p ][ 38 2.008956 007.1983.Octopussy.720p.hdtv.x264 ][ 42 2.844396 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 44 7.665204 The Secret of Moonarce [720p] ][ 47 1.648925 IMAX Ring of Fire [720p] [Documenter] ][ 55 9.240480 Meet The Parents    720p ][ 60 0.415329 Eragon 720p ][ 61 5.061433 G Force    720p ][ 71 4.666272 Eurotrip    720p ]
[ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 9 0.299931 Daredevil - 720p ][ 10 8.237954 Stargate-The Ark of Truth 720p ][ 18 0.410772 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 19 8.224514 Final Destination 4   1080p ][ 25 8.433877 Apollo 13   1080p ][ 31 6.296640 Darko 720P ][ 32 9.483675 Firewall 720p ][ 33 1.821950 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 35 7.382330 Gladiator 1080p ][ 38 2.008956 007.1983.Octopussy.720p.hdtv.x264 ][ 42 2.844396 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 44 7.665204 The Secret of Moonarce [720p] ][ 47 1.648925 IMAX Ring of Fire [720p] [Documenter] ][ 48 4.956958 Hulk 1    1080p ][ 49 5.400412 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 55 9.240480 Meet The Parents    720p ][ 60 0.415329 Eragon 720p ][ 61 5.061433 G Force    720p ][ 62 8.035188 Harold & Kumar   720p ][ 67 9.335771 Knowing 2009 720p ][ 70 2.180707 Hancock    1080p ][ 71 4.666272 Eurotrip    720p ][ 75 7.211695 Cloverfield   1080p ]
[ 1 3.427969 Kingdom of Heaven [2005] [720p] ][ 3 1.768690 My Bloody Valentine 3D    1080p ][ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 9 0.299931 Daredevil - 720p ][ 10 8.237954 Stargate-The Ark of Truth 720p ][ 11 2.345214 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 18 0.410772 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 21 8.377139 PBS ][ 22 5.940340 Jerry Maguire    1080p ][ 25 8.433877 Apollo 13   1080p ][ 31 6.296640 Darko 720P ][ 33 1.821950 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 35 7.382330 Gladiator 1080p ][ 42 2.844396 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 47 1.648925 IMAX Ring of Fire [720p] [Documenter] ][ 48 4.956958 Hulk 1    1080p ][ 49 5.400412 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 55 9.240480 Meet The Parents    720p ][ 59 4.214503 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 60 0.415329 Eragon 720p ][ 61 5.061433 G Force    720p ][ 62 8.035188 Harold & Kumar   720p ][ 65 7.675481 Starship Troopers 1  1080p ][ 70 2.180707 Hancock    1080p ][ 71 4.666272 Eurotrip    720p ][ 74 5.510212 High school musical 2006    720p ][ 75 7.211695 Cloverfield   1080p ][ 76 7.626986 Get Smart (2008) 1080p ][ 78 3.125565 Fantastic Four   1080p ]
[ 1 3.427969 Kingdom of Heaven [2005] [720p] ][ 3 1.768690 My Bloody Valentine 3D    1080p ][ 5 5.917003 Yes Man   720p ][ 6 5.079679 Crossing Over 720P ][ 7 5.554938 Crash    1080p ][ 9 0.299931 Daredevil - 720p ][ 10 8.237954 Stargate-The Ark of Truth 720p ][ 11 2.345214 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 18 0.410772 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 21 8.377139 PBS ][ 22 5.940340 Jerry Maguire    1080p ][ 25 8.433877 Apollo 13   1080p ][ 31 6.296640 Darko 720P ][ 33 1.821950 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 35 7.382330 Gladiator 1080p ][ 42 2.844396 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 47 1.648925 IMAX Ring of Fire [720p] [Documenter] ][ 48 4.956958 Hulk 1    1080p ][ 49 5.400412 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 56 2.962803 Afro samurai uncut 720p ][ 59 4.214503 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 60 0.415329 Eragon 720p ][ 61 5.061433 G Force    720p ][ 62 8.035188 Harold & Kumar   720p ][ 70 2.180707 Hancock    1080p ][ 74 5.510212 High school musical 2006    720p ][ 75 7.211695 Cloverfield   1080p ][ 76 7.626986 Get Smart (2008) 1080p ][ 78 3.125565 Fantastic Four   1080p ][ 81 5.442692 Jeniffer's Body ( 2009)    1080p ][ 83 0.188823 PIXAR Movie ][ 87 8.043926 Hero    1080p ][ 88 2.596885 Lord Of The Ring  - Return Of The King   720p ][ 91 7.485249 The Bank Job [720p] ][ 93 7.621348 Rocky 2 [1979] [BluRay] [720p] [AC3] ][ 100 9.749040 The Sniper 720p ]
